### ISSUE TITLE
NEI Crash

### ISSUE THREAD

**Issue by BlackSymbol** [MJRLegends/Space-Astronomy-Feedback-#192]
Crashing when trying to look up any recipe in NEI. Server was updated from 1.3.3 to 1.3.4 and then to 1.3.5. That crash happens to everybody playing on the server.
Log part associated with crash: http://pastebin.com/fsAiLZNQ

What I've tried:
- On server in "world" wiped the following folders (more precisely, data that was associated with me):
playerdata, stats, NEI, data, logisticspipes;
- Switching off different NEI modes (i.e. magnet, etc.);
- Single Player mode works fine (even for issue #188)

P.s. Doesn't seem to be related to #188.

**Comment by MJRLegends** [comment]
The log doesnt tell me anything i would need the full log

**Comment by BlackSymbol** [comment]
Sure, but I cannot upload it to pastebin (too large to process it seems). Therefore, GDrive: https://drive.google.com/file/d/0B9n0a1PeuvahT3VVdlIzWXU5TGM/view

**Comment by MJRLegends** [comment]
Ok thanks I will take a look later

**Comment by BlackSymbol** [comment]
Update: In client log file, I've found the same issue as in #188, so it seems that client tries to look up recipe, then crashes and then connection is forced to terminate what was called "by remote host" (i.e. by client). It seems NEI check things before showing up a recipe, that would explain it. Though I couldn't reproduce problem in SSP, so I don't know where the root of the problem is. I've tried also to downgrade Pneumaticraft (found at somebody's repo that there was some problem with it), but it didn't help.
